### ISSUE TITLE
Hiper type support

### ISSUE THREAD

**Issue by sibelius** [braintree/credit-card-type#59]
Based on this one https://github.com/moip/credit-card-validator/blob/master/src/main/java/br/com/moip/creditcard/HiperCreditCard.java

**Comment by crookedneighbor** [comment]
Closed as a duplicate of https://github.com/braintree/credit-card-type/issues/57

We must have official documentation before we can add additional card types to this library.

**Comment by crookedneighbor** [comment]
v6.2.0 allows you to add custom card brands. See https://github.com/braintree/credit-card-type#adding-card-types for documentation.
